pkg/packagekit/history.jsx

```diff
@@ -106,6 +106,7 @@ export class History extends React.Component {
 
         return (
             <ListingTable caption={_("Update History")}
+                          aria-label={_("Updates History")}
```

Comment:
Please use the same string. Also, perhaps ListingTable could just default to setting aria-label to caption if not specified? That should fix more cases in a more generic fashion.

Reply:
Why are we using caption here? Does it render to `<caption>`? If so, we need to switch to using a heading. If "caption" actually gets rendered out to a heading, like `<h3>`, then that's fine, but it should be consistent in `updates.jsx` too, as @martinpitt points out below.